### PR TITLE
Fix GCC 8 warning:

### PR DESCRIPTION
**Pull request by marxin**
/home/marxin/Programming/libyui/src/YDialog.cc: In static member function ‘static void YDialog::showText(const string&, bool)’:
/home/marxin/Programming/libyui/src/YDialog.cc:690:26: error: catching polymorphic type ‘class YUIException’ by value [-Werror=catch-value=]
     catch ( YUIException exception )
                          ^~~~~~~~~
/home/marxin/Programming/libyui/src/YDialog.cc: In static member function ‘static bool YDialog::showRelNotesText()’:
/home/marxin/Programming/libyui/src/YDialog.cc:814:26: error: catching polymorphic type ‘class YUIException’ by value [-Werror=catch-value=]
     catch ( YUIException exception )
                          ^~~~~~~~~

### CODE REVIEW

File: src/YDialog.cc [hunk 1]
```diff
@@ -687,7 +687,7 @@ YDialog::showText( const std::string & text, bool useRichText )
 	dialog->waitForEvent();
 	dialog->destroy();
     }
-    catch ( YUIException exception )
+    catch ( YUIException &exception )
```
Comment (aschnell) [inline src/YDialog.cc]: Is adding ```const``` possible?

**Comment by aschnell** [comment]
LGTM

**Comment by marxin** [comment]
Thanks for help, I've rebased the commits into a single one.

**Comment by aschnell** [comment]
Seems as if the version in the spec file is not automatically updated. Please do so manually.

**Comment by marxin** [comment]
@aschnell Thanks, done and now build system is happy.

**Comment by aschnell** [comment]
Thanks, LGTM.

**Comment by marxin** [comment]
@aschnell Great. Can you please submit the package to Factory?

**Comment by aschnell** [comment]
This should happen automatically but the CI complains about a missing bug/fate number. AFAIK this is (for factory) more restrictive than needed. So please make another PR which adds the bug number (#1084636) to the changelog.

**Comment by marxin** [comment]
Ah, done in #124.

**Pull request merged**